downstream/modules/platform/con-controller-metrics-monitor-controller.adoc

```diff
@@ -0,0 +1,29 @@
+[id="con-controller-metrics-monitor-controller"]
+
+= Metrics to monitor {ControllerName}
+
+Monitor your {ControllerName} hosts at the system and application levels. 
+
+System level monitoring includes the following information:
+
+* Disk I/O
+* RAM utilization
+* CPU utilization
+* Network traffic 
+
+Application level metrics provide data that the application knows about the system. This data includes the following information: 
+
+* How many jobs are running in a given instance
+* Capacity information about instances in the cluster
+* How many inventories are present 
+* How many hosts are in those inventories
+
+Using system and application metrics can help you identify what was happening in the application when a service degradation occurred. Information about {ControllerName}'s performance over time helps when diagnosing problems or doing capacity planning for future growth.
```

Comment:
Is there a reason the text switches to italics?

Reply:
I'm not sure! There's no indication in VS code as to why this is happening, and it doesn't show as italics when I build the docs.